test/jdk/javax/swing/ImageIcon/LoadInterruptTest.java

```diff
@@ -0,0 +1,171 @@
+<<<<<<< HEAD
+/*
+ * Copyright (c) 2022, Oracle and/or its affiliates. All rights reserved.
+ * DO NOT ALTER OR REMOVE COPYRIGHT NOTICES OR THIS FILE HEADER.
+ *
+ * This code is free software; you can redistribute it and/or modify it
+ * under the terms of the GNU General Public License version 2 only, as
+ * published by the Free Software Foundation.
+ *
+ * This code is distributed in the hope that it will be useful, but WITHOUT
+ * ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or
+ * FITNESS FOR A PARTICULAR PURPOSE.  See the GNU General Public License
+ * version 2 for more details (a copy is included in the LICENSE file that
+ * accompanied this code).
+ *
+ * You should have received a copy of the GNU General Public License version
+ * 2 along with this work; if not, write to the Free Software Foundation,
+ * Inc., 51 Franklin St, Fifth Floor, Boston, MA 02110-1301 USA.
+ *
+ * Please contact Oracle, 500 Oracle Parkway, Redwood Shores, CA 94065 USA
+ * or visit www.oracle.com if you need additional information or have any
+ * questions.
+ */
+/*
+ * @test
+ * @bug 8236987
+ * @summary  Verifies if Print Function is removed and LoadStatus is changed to ABORTED when interrupted/ still LOADING.
+ * @run main LoadInterruptTest
+ */
```

Comment:
I would appreciate if you put this test description before the class declaration.

Reply:
> For thread safety, you should put all your code into `LoadAndSimulate`

I'm calling only one method i.e., setUpOutput()  from LoadAndSimulate, I think it'll be in the same thread, so its fine I guess.